src/lib/hips_handler.py

```diff
@@ -16,6 +16,11 @@ class HipsSurvey:
 
 
 def open_hips(hips_survey):
+    """
+    This is opens the hips survey
```

Comment:
ahaha maybe we should change this, this was left over when I was talking to you about docstrings right? or we can leave it and fix it up later

Reply:
Lmao ill remove